test/correctness/matlab.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Halide.h"
```

Comment:
This could be a generator test, and the aot test stub could verify that the pipeline still works as expected, and that the symbol at least exists for the matlab entry point (perhaps by extern declaring it and asserting its address is non-NULL? That should at least produce a linker error if the symbol isn't there)